support/carbon/carbon.go

```diff
@@ -170,237 +168,152 @@ const (
 )
 
 type Clock struct {
-	testNow bool
-	time    Carbon
+	testNow  bool
+	testTime Carbon
+	timezone string
 }
 
-var once sync.Once
-
-var internalClock *Clock
-
-// getInstance Get a singleton Clock object.
-func getInstance(timezone ...string) *Clock {
-	once.Do(func() {
-		internalClock = &Clock{
-			testNow: false,
-			time:    carbon.Now(timezone...),
-		}
-	})
-
-	return internalClock
-}
+var clock = &Clock{}
 
 // SetTestNow Set the test time. Remember to unset after used.
-func SetTestNow(testNow Carbon) {
-	t := getInstance()
-	t.testNow = true
-	t.time = testNow
+func SetTestNow(testTime Carbon) {
+	clock.testNow = true
+	clock.testTime = testTime
 }
 
 // UnsetTestNow Unset the test time.
 func UnsetTestNow() {
-	t := getInstance()
-	t.testNow = false
-	t.time = carbon.Now()
+	clock.testNow = false
 }
 
 // IsTestNow Determine if the test now time is set.
 func IsTestNow() bool {
-	t := getInstance()
+	return clock.testNow
+}
 
-	return t.testNow
+// SetTimezone sets timezone.
+// 设置时区
+func SetTimezone(timezone string) {
+	clock.timezone = timezone
 }
 
 // Now return a Carbon object of now.
 func Now(timezone ...string) Carbon {
-	t := getInstance(timezone...)
-
-	if t.testNow {
-		return t.time
+	if IsTestNow() {
+		return clock.testTime
 	}
 
-	now := carbon.Now(timezone...)
-	t.time = now
```

Comment:
I think the `time` just store test time is ok, we don't need to set it in every function, so I remove it.